pmm/pmm2-submodules-rewind.groovy

```diff
@@ -16,15 +16,15 @@ pipeline {
             steps {
                 deleteDir()
 
-                git branch: 'PMM-2.0', credentialsId: 'GitHub SSH Key', poll: false, url: 'git@github.com:Percona-Lab/pmm-submodules'
+                // git branch: 'PMM-2.0', credentialsId: 'GitHub SSH Key', poll: false, url: 'git@github.com:Percona-Lab/pmm-submodules'
                 
                 withCredentials([sshUserPrivateKey(credentialsId: 'GitHub SSH Key', keyFileVariable: 'SSHKEY', passphraseVariable: '', usernameVariable: '')]) {
                     sh '''
+                        git clone --single-branch --branch "PMM-2.0" https://github.com/Percona-Lab/pmm-submodules .
```

Comment:
This fixes the failure of Jenkins' native git tool (reason of failure is unknown so far). Check [slack](https://perconacorp.slack.com/archives/G4VC14WTF/p1707062344085489) for more details.